libinitializer/LedgerInitializer.cpp

```diff
@@ -32,56 +32,59 @@ using namespace dev::initializer;
 
 void LedgerInitializer::initConfig(boost::property_tree::ptree const& _pt)
 {
-    namespace fs = boost::filesystem;
     INITIALIZER_LOG(DEBUG) << LOG_BADGE("LedgerInitializer") << LOG_DESC("initConfig");
-    m_groupDataDir = _pt.get<string>("group.group_data_path", "data/");
-    auto groupConfigPath = _pt.get<string>("group.group_config_path", "conf/");
+
     assert(m_p2pService);
+    m_groupDataDir = _pt.get<string>("group.group_data_path", "data/");
+    m_groupConfigPath = _pt.get<string>("group.group_config_path", "conf/");
```

Comment:
I think `assert(m_groupConfigPath.size() != 0)` should be here.

Reply:
done